### PR TITLE
INTMDB-455: bi_connector settings in mongodbatlas_advanced_cluster fix

### PR DESCRIPTION
**Pull request by martinstibbe**
## Description

bi_connector settings in mongodbatlas_advanced_cluster fix to make state match connector setting

Link to any related issue(s):

## Type of change:

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update
- [x] Documentation fix/enhancement

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have read the [contribution guidelines](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CONTRIBUTING.md)
- [ ] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run make fmt and formatted my code

### CODE REVIEW

**Review by Zuhairahmed — APPROVED** [review]
LGTM! thanks @martinstibbe for getting this one in too!

File: mongodbatlas/resource_mongodbatlas_advanced_cluster_migrate.go [hunk 1]
```diff
@@ -0,0 +1,245 @@
+package mongodbatlas
+
```
Comment (Zuhairahmed) [inline mongodbatlas/resource_mongodbatlas_advanced_cluster_migrate.go:2]: @martinstibbe do we need a _test.go file as well as part of this new supporting resource?

Comment (andreaangiolillo) [reply mongodbatlas/resource_mongodbatlas_advanced_cluster_migrate.go:2]: +1

File: mongodbatlas/resource_mongodbatlas_advanced_cluster_migrate.go [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+import (
+	"bytes"
+	"context"
+	"time"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+)
+
+func resourceMongoDBAtlasAdvancedClusterResourceV0() *schema.Resource {
```
Comment (andreaangiolillo) [inline mongodbatlas/resource_mongodbatlas_advanced_cluster_migrate.go:12]: [q] What is the `V0` in the name stand for?

Comment (martinstibbe) [reply mongodbatlas/resource_mongodbatlas_advanced_cluster_migrate.go:12]: V0 schema part of state migration function to move old schema value to new one bi_connector becomes bi_connector_config in this release breaking change

**Review by andreaangiolillo — APPROVED** [review]
LGTM

**Pull request merged**